src/runtime/containerd-shim-v2/shim_management.go

```diff
@@ -24,15 +24,14 @@ import (
 	dto "github.com/prometheus/client_model/go"
 	"github.com/prometheus/common/expfmt"
 
-	"github.com/sirupsen/logrus"
-
 	"google.golang.org/grpc/codes"
 
 	mutils "github.com/kata-containers/kata-containers/src/runtime/pkg/utils"
 )
 
 var (
 	ifSupportAgentMetricsAPI = true
+	shimManagementLog        = shimLog.WithField("subsystem", "shim_management")
```

Comment:
- This is rather a long variable name. How about `shimMgtLog`?
- `s/shim_management/shim-management/`